RFC-003-SWAP-Basic.md

```diff
@@ -183,6 +185,43 @@ When Bob learns the secret from Alice's redeem activation of β-HTLC he MUST act
 He MUST make sure he does this before `alpha_expiry` or risks both losing **B** and not gaining **A**.
 To activate the redeem path he uses the secret and the procedure defined in the specification of the α-HTLC.
 
+## Expiry Time Considerations
+
+Conceptually there is a time window from when the HTLC is deployed until the HTLC expires and can be refunded.
+The protocol depends on the alpha time window being a superset of the beta time window.
+
+
+  time ->
+
+     deploy                                 expiry
+
+Alpha  |----------------------------------------|
+
+Beta            |--------------------|
+
+             deploy              expiry
+
+In addition, in order for the swap to be atomic the alpha asset must be redeemed *before* `beta_expiry`.
+To be more precise; the alpha redeem transaction must have been accepted into the alpha ledger before `beta_expiry`.
+This means that the window of time for which atomicity is guaranteed for Alice is actually *smaller* than it at first appears.
+There exists a point on the beta time window depicted above (and again below) after which a redeem transaction may not get included into the ledger before the expiry time, at which time it is possible for Bob to attempt a refund transaction.
+This point is marked below with `???` because it is dependant on the ledger.
+
+
```

Comment:
📣   I think you want to make this code-block. Please add backticks ``` before and after.

Reply:
Thanks, removed.